extensions/amp-story/amp-story.md

```diff
@@ -785,6 +785,62 @@ You can apply multiple entrance animations on one element (for example, an eleme
 If a composed animation is supposed to start after the end of a separate element's animation, make sure that all nested elements that compose the animation have the attribute `animate-in-after` set to the same `id`.
 {% endcall %}
 
+## Branching
+
+Stories support fragment paramenters in the url. By utilizing this, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story. An example is a table of contents.
+
+### Manipulating Navigation
+
+The feature allows you to manipulate navigation within a story. When using it, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. If a user navigates from `page-1` to `page-5` and then `page-6`, navigating backwards will follow this exact path (`page-6` to `page-5` and finally `page-1`). The other pages will not be exposed to the user.
```

Comment:
I'd update to:

Branching allows navigation manipulate within a story. The story tracks navigation. If a user navigates from `page-1` to `page-5` and then `page-6`, navigating backwards will follow the exact path. The skipped pages are not exposed to the user when navigating backwards, it will follow `page-6`, to `page-5`, and ends back at `page-1`.

Reply:
Done. I changed your manipulate to manipulation because I think this is what you meant, but let me know if I'm wrong :)